gemini/sample-apps/image-bash-jam/tts.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+##############################################################################
+# word to text via CURL :)
+# Docs https://cloud.google.com/text-to-speech/docs/create-audio-text-command-line
+# Before you run, pls execute:
+#
+# 	gcloud auth application-default login
+# 	gcloud auth application-default set-quota-project "$PROJECT_ID"
+#
+##############################################################################
+
+. .envrc
+
+set -euo pipefail
+
+#PROJECT_ID=$(gcloud config get-value project)
+#SENTENCE="$(echo "$@" | sed "s/'/\\\\'/g")" # c'e' l'uomo => "c e l uomo"
+SENTENCE="${*//\'/\\\'}"        # c'e' l'uomo => "c\' e l\'uomo"
+TMP_OUTPUT_FILE='.tmp.tts-output.json'
+JQ_PATH='.audioContent'
+DEFAULT_LANG="en-US"
+TTS_LANG="${TTS_LANG:-$DEFAULT_LANG}"
```

Comment:
Would it be possible to set the default voice to Journey `
en-US-Journey-D` (en-US only) or Studio `en-US-Studio-O`? 

They have the latest TTS models.

List of models: https://cloud.google.com/text-to-speech/docs/voices

Reply:
Will do. 
Let me try to see if this breaks anything.

Couple of notes:
1. following your suggestion makes this script hard to maintain: while on Jan24 this might be the latest, what about 1 year from now? I'd rather us en-US and trust PM to make it converge to latest when its safe enough.

2. I tried to change the DEFAULT_LANG abd I get this error:
```
{
  "error": {
    "code": 400,
    "message": "Voice '' does not exist. Is it misspelled?",
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.DebugInfo",
        "detail": "[ORIGINAL ERROR] generic::invalid_argument: Voice '' does not exist. Is it misspelled? [google.rpc.error_details_ext] { message: \"Voice \\'\\' does not exist. Is it misspelled?\" }"
      }
    ]
  }
}
```

Happy for you to propose a fix s i dont see an easy way out here.

Reply:
Ah! I think I see what the issue is.

The language code should still be set as `en-US` but there's another parameter in `voice` for `name`.

https://cloud.google.com/text-to-speech/docs/reference/rest/v1/VoiceSelectionParams

Example:

```json
  'voice': {
      'languageCode': '$TTS_LANG',
      'ssmlGender': '$DEFAULT_GENDER'
      'name': '$TTS_VOICE'
    },
```

Reply:
OOOHHH so it would be 
* DEFAULT_LANG  stays en-US
* voice::name = en-US-Journey-D 
* ?